lib/developer_portal/lib/liquid/drops/provider.rb

```diff
@@ -155,6 +155,10 @@ def multiple_users_allowed?
         @model.settings.multiple_users.visible?
       end
 
+      def account_plans_allowed?
```

Comment:
This is not documented.

Reply:
Right, pushed the documentation @hallelujah